lib/cucumber/formatter/html.rb

```diff
@@ -565,7 +565,9 @@ class NullConverter; def convert(code, _pre); code; end; end #:nodoc:
         def snippet(error)
           raw_code, line = snippet_for(error[0])
           highlighted = @@converter.convert(raw_code, false)
-          highlighted << "\n<span class=\"comment\"># gem install syntax to get syntax highlighting</span>" if @@converter.is_a?(NullConverter)
+
+          highlighted += "\n<span class=\"comment\"># gem install syntax to get syntax highlighting</span>" if @@converter.is_a? NullConverter
```

Comment:
Question: Can change be made smaller by not changing the use of parentheses? Does that change anything in warnings output or otherwise?

Reply:
Yep - I'll do this, too. It has no impact on anything at all. :)